index.ts

```diff
@@ -0,0 +1,27 @@
+'use strict'
+
+import { Config } from './config'
+
+/**
+* Start the profiling agent
+* Currently unimplemented
+*
+* @param {object=} config - Profiler configuration
+*
+* @example
+* profiler.start();
+*/
+function start(projectConfig: Config) {
+  console.log("UNIMPLEMENTED");
```

Comment:
Is using `console.log` how people log things in Node.js? I'd expect there is something more specialized.